src/kepler/ipc.zig

```diff
@@ -171,11 +171,11 @@ pub const NoteQueue = struct {
     fn send(self: *@This(), note: *Note) !void {
         // Check queue state for the first time
         if (@atomicLoad(State, &self.state, .Acquire) != .Up) {
-            return error.IDK;
+            return error.Unreachable;
```

Comment:
What's `error.Unreachable` supposed to mean? `unreachable;` or something else?

Reply:
Unreachable error means that other peer in the stream has closed the connection abd won't get a notification. So no, this code is actually reachable.

Reply:
Oh, so it's "Unreachable" as in the peer is unreachable, and not as in that the code is?

Reply:
Yep

Reply:
What about a name like `ConnectionAborted` or `PeerUnreachable` then?